src/CoreArbiterServer.h

```diff
@@ -103,6 +108,7 @@ class CoreArbiterServer {
 
         CoreInfo(int id, std::string managedTasksPath)
             : id(id)
+            , hypertwin_id(PerfUtils::Util::getHyperTwin(id))
```

Comment:
Is there a pull request for the PerfUtils package (or has getHyperTwin always been present there)?